### PR TITLE
mixins/observable: Update "Observing Property Changes" comment section

### PR DESCRIPTION
**Pull request by Turbo87**

### CODE REVIEW

File: packages/ember-runtime/lib/mixins/observable.js [hunk 1]
```diff
@@ -57,8 +57,9 @@ import isNone from 'ember-metal/is_none';
 
   ## Observing Property Changes
 
-  You typically observe property changes simply by adding the `observes`
-  call to the end of your method declarations in classes that you write.
+  You typically observe property changes simply by using the the `Ember.observer`
```
Comment (tricknotes) [inline packages/ember-runtime/lib/mixins/observable.js]: "the the" -> "the"?

Comment (Turbo87) [inline packages/ember-runtime/lib/mixins/observable.js]: oh, missed that. thanks!

**Comment by rwjblue** [comment]
Can you squash and prefix with '[DOC release]'?

**Comment by Turbo87** [comment]
@rwjblue done

**Pull request merged**